### PR TITLE
Quick text fixes, and small about page button fix

### PR DESCRIPTION
**Pull request by qdang16**

### CODE REVIEW

File: app/views/apply/students.html.slim [hunk 1]
```diff
@@ -55,8 +55,8 @@
               span.circle
             .timeline-content
               p
-                | A fun little event for you guys
-                | to flex your problem solving muscles in a team of other applicants. Second round
+                | A fun little event for you all
```
Comment (chenhoward) [inline app/views/apply/students.html.slim]: Nice catch

Comment (triciasfu) [inline app/views/apply/students.html.slim]: @qdang16 ++

Comment (qdang16) [inline app/views/apply/students.html.slim]: ;)

**Comment by qdang16** [comment]
did hound die on me

**Comment by SamLau95** [comment]
Thanks Quinton! The issue with leaving trailing spaces in the slim file is that my sublime and other people's if they've configured it will trim all the spaces once the file gets saved. Why not instead pull the text into `en.yml` and then use i18n on this page? You can see https://github.com/calblueprint/calblueprint.org/blob/master/app/views/apply/nonprofits.html.slim and https://github.com/calblueprint/calblueprint.org/blob/master/config/locales/en.yml#L60-L118

**Comment by SamLau95** [comment]
Thanks @qdang16 ! :)

**Pull request merged**